README.md

```diff
@@ -42,6 +42,13 @@ For pre-requisites specific to Terraform CLI and ORM, see their corresponding se
 
 ## Create the Stack Using the OCI Marketplace
 
+The WebLogic Server for OCI applications available in the Marketplace are:
```

Comment:
I don't think this is a blocker, but technically the Image offerings are considered "applications" in Marketplace. I understand that the ## Create the Stack Using the OCI Marketplace already give the context that this is a stack, but I'm still thinking that maybe this should be "WebLogic Server for OCI stack applications". Since I'm not convinced I'm right about changing this, if you have doubts, consult with Adrian.

Reply:
You are right. I added the word stack as you suggested